components/admin/lessons/AdminLessonInputs.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react'
+import { useAddModuleMutation } from '../../../graphql'
+import { formChange } from '../../../helpers/formChange'
+import { FormCard, Option } from '../../FormCard'
+import { AlertFillIcon, CheckCircleIcon } from '@primer/octicons-react'
+import styles from '../../../scss/adminLessonInputs.module.scss'
+import { Spinner } from 'react-bootstrap'
+import { GraphQLError } from 'graphql-request/dist/types'
+
+type Props = {
+  lesson: { title: string; id: number }
+}
+
+enum Error {
+  INVALID_DATA = 'missing module name or description'
+}
+
+const values: Option[] = [
+  {
+    title: 'Module Name'
+  },
+  {
+    title: 'Description',
+    type: 'MD_INPUT'
```

Comment:
please use the constant instead
https://github.com/garageScript/c0d3-app/blob/master/components/FormCard.tsx#L13